docker/README.md

```diff
@@ -32,16 +32,18 @@ The base flavor contains PyTorch pre-installed; the vision flavor also includes
-| Ubuntu 20.04   | Base     | 1.11.0            | 11.5.2         | 3.8              | `mosaicml/pytorch:1.11.0_cu115-python3.8-ubuntu20.04`                                            |
-| Ubuntu 20.04   | Base     | 1.11.0            | cpu            | 3.8              | `mosaicml/pytorch:1.11.0_cpu-python3.8-ubuntu20.04`                                              |
-| Ubuntu 20.04   | Base     | 1.10.2            | 11.3.1         | 3.7              | `mosaicml/pytorch:1.10.2_cu113-python3.7-ubuntu20.04`                                            |
-| Ubuntu 20.04   | Base     | 1.10.2            | cpu            | 3.7              | `mosaicml/pytorch:1.10.2_cpu-python3.7-ubuntu20.04`                                              |
-| Ubuntu 20.04   | Vision   | 1.12.1            | 11.6.2         | 3.9              | `mosaicml/pytorch_vision:latest`, `mosaicml/pytorch_vision:1.12.1_cu116-python3.9-ubuntu20.04`   |
-| Ubuntu 20.04   | Vision   | 1.12.1            | cpu            | 3.9              | `mosaicml/pytorch_vision:latest_cpu`, `mosaicml/pytorch_vision:1.12.1_cpu-python3.9-ubuntu20.04` |
+| Linux Distro   | Flavor   | PyTorch Version   | CUDA Version   | Python Version   | Docker Tags                                                                                       |
+|----------------|----------|-------------------|----------------|------------------|---------------------------------------------------------------------------------------------------|
+| Ubuntu 20.04   | Base     | 1.13.0            | 11.7.1         | 3.10             | `mosaicml/pytorch:latest`, `mosaicml/pytorch:1.13.0_cu117-python3.10-ubuntu20.04`                 |
+| Ubuntu 20.04   | Base     | 1.13.0            | cpu            | 3.10             | `mosaicml/pytorch:latest_cpu`, `mosaicml/pytorch:1.13.0_cpu-python3.10-ubuntu20.04`               |
+| Ubuntu 20.04   | Base     | 1.12.1            | 11.6.2         | 3.9              | `mosaicml/pytorch:1.12.1_cu116-python3.9-ubuntu20.04`                                             |
+| Ubuntu 20.04   | Base     | 1.12.1            | cpu            | 3.9              | `mosaicml/pytorch:1.12.1_cpu-python3.9-ubuntu20.04`                                               |
+| Ubuntu 20.04   | Base     | 1.11.0            | 11.5.2         | 3.8              | `mosaicml/pytorch:1.11.0_cu115-python3.8-ubuntu20.04`                                             |
+| Ubuntu 20.04   | Base     | 1.11.0            | cpu            | 3.8              | `mosaicml/pytorch:1.11.0_cpu-python3.8-ubuntu20.04`                                               |
+| Ubuntu 20.04   | Base     | 1.10.2            | 11.3.1         | 3.7              | `mosaicml/pytorch:1.10.2_cu113-python3.7-ubuntu20.04`                                             |
+| Ubuntu 20.04   | Base     | 1.10.2            | cpu            | 3.7              | `mosaicml/pytorch:1.10.2_cpu-python3.7-ubuntu20.04`                                               |
```

Comment:
Can we take this opportunity to get rid of the Torch 1.10.2 images?  Would like to actively support 3 versions, these will still be around they just won't update anymore.